e2e/summary-action-bar.e2e-spec.ts

```diff
@@ -101,7 +101,7 @@ describe('Summary Action Bar', () => {
   it('should match previous tab with a summary action bar screenshot', (done) => {
     SkyHostBrowser.get('visual/tab-summary-action-bar');
     SkyHostBrowser.setWindowBreakpoint('lg');
-    element(by.css('#sky-tab-2-nav-btn')).click();
+    element(by.css('#sky-tab-1-nav-btn')).click();
```

Comment:
This fixes what seems to be a broken test due to the tab id's switching to using a zero based index. I'll have to follow up with this in stand up if not here.